commands/variable/list/list.go

```diff
@@ -0,0 +1,92 @@
+package list
+
+import (
+	"github.com/MakeNowJust/heredoc"
+	"github.com/profclems/glab/api"
+	"github.com/profclems/glab/commands/cmdutils"
+	"github.com/profclems/glab/internal/glrepo"
+	"github.com/profclems/glab/pkg/iostreams"
+	"github.com/profclems/glab/pkg/tableprinter"
+	"github.com/spf13/cobra"
+	"github.com/xanzy/go-gitlab"
+)
+
+type ListOpts struct {
+	HTTPClient func() (*gitlab.Client, error)
+	IO         *iostreams.IOStreams
+	BaseRepo   func() (glrepo.Interface, error)
+
+	ValueSet bool
+	Group    string
+}
+
+func NewCmdSet(f *cmdutils.Factory, runE func(opts *ListOpts) error) *cobra.Command {
+	opts := &ListOpts{
+		IO: f.IO,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "list",
+		Short:   "List project or group variables",
+		Aliases: []string{"new", "create"},
+		Args:    cobra.ExactArgs(0),
+		Example: heredoc.Doc(`
+			$ glab variable list
+		`),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			// Supports repo override
+			opts.HTTPClient = f.HttpClient
+			opts.BaseRepo = f.BaseRepo
+
+			if runE != nil {
+				err = runE(opts)
+				return
+			}
+			err = listRun(opts)
+			return
+		},
+	}
+
+	cmd.Flags().StringVarP(&opts.Group, "group", "g", "", "List group variables")
+
+	return cmd
+}
+
+func listRun(opts *ListOpts) error {
+	//c := opts.IO.Color()
```

Comment:
You have left some commented-out code right here.

Reply:
Also, right  now I'm confused with what to include in the table; whether to include the value of the variable